manifests/init.pp

```diff
@@ -33,6 +33,8 @@
   $epel_includepkgs                       = undef,
   $epel_sslclientkey                      = undef,
   $epel_sslclientcert                     = undef,
+  $epel_username                          = undef,
```

Comment:
Can you please add a datatype for the new parameters? I think something like this will work fine:
```suggestion
  Optional[String[1]] $epel_username      = undef,
```

Reply:
Done